crates/cubecl-core/src/runtime_tests/file.rs

```diff
@@ -0,0 +1,56 @@
+use crate::{self as cubecl};
+use cubecl::prelude::*;
+use cubecl_common::bytes::Bytes;
+use std::{io::Write, time::SystemTime};
+
+const MB: usize = 1024 * 1024;
+pub fn test_file_memory<R: Runtime>(client: ComputeClient<R::Server>) {
+    let now = SystemTime::now();
+    let duration = now.duration_since(SystemTime::UNIX_EPOCH).unwrap();
+    let file_name = format!("/tmp/{:?}", duration);
+    let data_init = (0i32..MB as i32).collect::<Vec<i32>>();
```

Comment:
Recommend using tempfile lib to create manage temp files like in https://github.com/tracel-ai/burn/blob/main/crates/burn-dataset/src/dataset/sqlite.rs#L713

`use tempfile::{NamedTempFile, TempDir, tempdir};`